Cesium.Runtime/VoidPtr.cs

```diff
@@ -3,14 +3,14 @@ namespace Cesium.Runtime;
 /// <summary>A class encapsulating an opaque pointer (aka <code>void*</code> in C).</summary>
 public readonly unsafe struct VoidPtr
 {
-    private readonly long _value;
+    private readonly IntPtr _value;
```

Comment:
No, this is not correct. `VoidPtr` is supposed to have a size of `long` and not `IntPtr`: 8 bytes on every system. It is used for portable code, for the "wide arch".

For any other arch, we just use normal pointers.